setup.cfg

```
@@ -18,9 +18,12 @@ classifiers =
     Topic :: Internet :: WWW/HTTP
     Framework :: Django
     Framework :: Django :: 3.2
-    Framework :: Django :: 4
-    Framework :: Wagtail :: 2
-    Framework :: Wagtail :: 3
+    Framework :: Django :: 4.0
+    Framework :: Django :: 4.1
+    Framework :: Wagtail :: 2.15
+    Framework :: Wagtail :: 2.16
+    Framework :: Wagtail :: 3.0
+    Framework :: Wagtail :: 4.0
```

```suggestion
    Framework :: Wagtail :: 2
    Framework :: Wagtail :: 3
    Framework :: Wagtail :: 4
```

Please always check https://pypi.org/search/?c=Framework+%3A%3A+Wagtail (see the options in sidebar)

---

Thanks I will update it. Good call 👍